tests/utils/mock-api.js

```diff
@@ -15,11 +15,7 @@ const startMockApi = ({ routes }) => {
   const requests = []
   const app = express()
   app.use(bodyParser.urlencoded({ extended: true }))
-  app.use(
-    bodyParser.json({
-      type: () => true,
```

Comment:
We can remove the following as per - https://github.com/netlify/cli/pull/2462#discussion_r638850144